service_storagefacility/app/controllers/StorageController.scala

```diff
@@ -484,8 +484,8 @@ final class StorageController @Inject()(
   def search(
       mid: Int,
       searchStr: Option[String],
-      page: Int = 1,
```

Comment:
I cant see any route changes that can legitimate this change. This could break if the client (uncorrectly) uses the endpoint without limit or page. Optionally you could move the default values to routes?